gtf2featureAnnotation.R

```diff
@@ -143,13 +143,24 @@ if (is.na(opt$output_file)){
 # Try to get some annotation fields from the the transcript names themselves-
 # this will likely not work for non-Ensembl GTFs
 
-parse_ensembl_fasta_transcript_info <- function(tname){
-  description <- sub(".*description:(.*)", "\\1", tname)
-  tname <- sub(" description:.*", '', tname)
+parse_fasta_transcript_info <- function(tname, fa_header_style=NULL){
+  if (fa_header_style == 'ensembl'){
+    sep <- ':'
+  }else if (fa_header_style == 'wormbase'){
+    sep <- '='
+  }else{
+    sep <- ':|='  
+  }
+  
+  description=NULL
+  if (grepl('description:', tname)){
+    description <- sub(".*description:(.*)", "\\1", tname)
+    tname <- sub(" description:.*", '', tname)
+  }
```

Comment:
I think this is only run when `fa_header_style == 'ensembl'`, so you will not define variable `description` for Wormbase cases (it will be `NULL`). Is that correct?

Reply:
Correct, and then the description won't be added to the transcirpt-derived info futher down